databuilder/extractor/bigquery_metadata_extractor.py

```diff
@@ -63,12 +63,14 @@ def _retrieve_tables(self, dataset):
 
                 # BigQuery tables also have interesting metadata about partitioning
                 # data location (EU/US), mod/create time, etc... Extract that some other time?
-                schema = table['schema']
                 cols = []
-                if 'fields' in schema:
-                    total_cols = 0
-                    for column in schema['fields']:
-                        total_cols = self._iterate_over_cols('', column, cols, total_cols + 1)
+                # Not all tables have schemas
+                if 'schema' in table:
```

Comment:
do you need to do the same for bigquery_usage extractor as well bigquery_watermark_extracotr?

Reply:
I mean https://github.com/lyft/amundsendatabuilder/blob/master/databuilder/extractor/bigquery_usage_extractor.py and https://github.com/lyft/amundsendatabuilder/blob/master/databuilder/extractor/bigquery_watermark_extractor.py

Reply:
I don't see anywhere the schema is being referenced. I haven't started using those modules yet.